src/test/java/io/openliberty/tools/intellij/it/SingleModMPProjectTestCommon.java

```diff
@@ -923,6 +924,69 @@ public static void prepareEnv(String projectPath, String projectName) {
                 "prepareEnv. Exit. ProjectName: " + projectName);
     }
 
+    /**
+     * Tests that when a Liberty run configuration is removed, any custom start parameters
+     * linked to the Liberty module are also cleared.
+     */
+    @Test
+    @Video
+    public void testCustomStartParametersClearedOnConfigRemoval() {
+        String testName = "testCustomStartParametersClearedOnConfigRemoval";
+        String absoluteWLPPath = Paths.get(getProjectsDirPath(), getSmMPProjectName(), getWLPInstallPath()).toString();
+
+        // Remove all existing configurations for a clean state.
+        UIBotTestUtils.deleteLibertyRunConfigurations(remoteRobot);
+
+        // Trigger the start with parameters configuration dialog.
+        UIBotTestUtils.runLibertyActionFromLTWDropDownMenu(remoteRobot, "Start...", true, 3);
+
+        // Run the configuration dialog.
+        UIBotTestUtils.runStartParamsConfigDialog(remoteRobot, getStartParamsDebugPort());
+
+        try {
+            // Validate that the project started.
+            TestUtils.validateProjectStarted(testName, getSmMpProjResURI(), getSmMpProjPort(), getSmMPProjOutput(), absoluteWLPPath, false);
+
+            // To check if debug port is set to a custom value (e.g., 9876)
+            TestUtils.checkDebugPort(absoluteWLPPath, 9876);
+
+        } catch (IOException e) {
+            System.err.println("Error reading the file: " + e.getMessage());
```

Comment:
```suggestion
            System.err.println("Error reading the server.env file: " + e.getMessage());
```

Reply:
Instead of a print line statement, we should fail the test if we cannot read the `server.env`. You can use `Assertions.fail()`.

Reply:
> Instead of a print line statement, we should fail the test if we cannot read the `server.env`. You can use `Assertions.fail()`.

Changed to Assertions.fail()